website_event_snippet_calendar/models/event_event.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Tecnativa S.L. - Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.addons.mail.models.mail_template import format_tz
+from odoo import fields, models, api
+
+
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    # Located with predictable format
+    date_begin_pred_located = fields.Char(
+        compute='_compute_date_begin_pred_located')
```

Comment:
```suggestion
        compute='_compute_date_begin_pred_located',
        store=True,
    )
```

If you load lots of events, this will make a difference.